static/scss/user_page.scss

```diff
@@ -0,0 +1,56 @@
+.user-page {
+  .avatar {
+    position: relative;
+    span.img{
+      width: 38px;
+      height: 38px;
+      border-radius: 50%;
+      background-color: rgba(0, 0, 0, 0.3);
+      position: absolute;
+      top: 0;
+      right: 17px;
+      text-align: center;
+      line-height: 35px;
+      img{
+        width: 22px;
+        height: auto;
+      }
+    }
+  }
+  .profile-title {
+    margin-top: 19px;
+    font-size: 25px;
+    color: black;
+    font-weight: 500;
+  }
+  .profile-company-name {
+    font-size: 18px;
+    color: #98989A;
+    padding: 15px 0;
+  }
+  .profile-email {
+    color: #0E52CE;
+    font-size: 18px;
+    margin-left: -5px;
+  }
+  .email-icon {
+    margin-right: 8px;
+    width: 28px;
+    height: 28px;
+    vertical-align: middle;
+  }
+  .edit-profile-holder {
+    width: 100%;
+    text-align: right;
+  }
+  .user-form-grid {
+    display: flex;
+    flex-direction: row;
+    .column {
+      display: flex;
+      flex-direction: column;
+      width: 250% !important;
+      margin-left: 15px;
+    }
+  }
+}
```

Comment:
general scss style stuff:

we generally try to leave a blank line after closing a block and before opening a new one. So instead of

```scss
.class {
  display: none;
  .other-class {
    color: purple;
  }
  .third-class {
    text-decoration: underline;
  }
}
```

we generally prefer:

```scss
.class {
  display: none;
  
  .other-class {
    color: purple;
  }
  
  .third-class {
    text-decoration: underline;
  }
}
```

when closing multiple blocks (like in the example when we close the `.third-class` and `.class` blocks) the curly braces don't need a blank line in between.

check out `page-layout.scss` and `user-chip.scss` for examples of this formatting.